### PR TITLE
Update Readme with instructions of integrating and using jupyter lite xblock with openedx

### PR DESCRIPTION
**Pull request by Abdul-Muqadim-Arbisoft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Update course advanced settings by adding `jupyterlite_xblock` as shown in below image and save changes 
+
+![Update settings image](https://github.com/edly-io/jupyterlite-xblock/blob/master/docs/images/update-settings.png?raw=True)
+
+Now you can access the jupyterlite xblock feature in the advanced component of course unit
+
+
+#### **Update settings of Jupyter Component**
+
+AI Coach Xblock will be available in Advanced component of course unit now. Add "JupyterLite" xblock in unit and click "Edit" button
```
Comment (ziafazal) [inline README.md]: AI Coach Xblock or Jupyter Xblock?

**Review by ziafazal — COMMENTED** [review]
@Abdul-Muqadim-Arbisoft thanks overall looks good just one change is needed

File: README.md (hunk 1 above)
Comment (Abdul-Muqadim-Arbisoft) [reply README.md]: Readme adjusted

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**